CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This release adds `Clone` and `Copy` implementations.
 
 ### [Added]
 
+- add `Methods::merge_replace` ([#1058](https://github.com/paritytech/jsonrpsee/pull/1058))
```

Comment:
nit: This is not necessarily needed now. We'll generate the changelog with the next release, since its not part of the v0.16.2, but of the next release